tools/start-local-vm

```diff
@@ -80,6 +83,8 @@ Options:
                         private partition will be lost
     --firmware-code     override the default firmware executable file
     --firmware-vars     override the initial firmware variable storage file
+    --boot-image-size   resize the boot disk image to the given size (e.g. 4096M)
+    --data-image-size   reisze the data disk image to the given size (e.g. 20G)
```

Comment:
```suggestion
    --data-image-size   resize the data disk image to the given size (e.g. 20G)
```

Reply:
Ran a spell checker on the script and fixed another typo